apps/visualization/utils.py

```diff
@@ -111,20 +113,28 @@ def get_outbreaks(iso3):
 @sync_to_async
 def get_country_indicators(iso3, outbreak, type):
     from .types import CountryIndicatorType
+    active_outbreaks = list(
+        Outbreaks.objects.filter(active=True).values_list('outbreak', flat=True)
+    )
+    qs = CountryFilterOptions.objects.filter(
+        emergency__in=active_outbreaks
+    )
```

Comment:
No need to pull and send it back to the database. Let's just use a query directly.
```suggestion
    active_outbreaks_qs = Outbreaks.objects.filter(active=True).values('outbreak')
    qs = CountryFilterOptions.objects.filter(
        emergency__in=active_outbreaks_qs
    )
```

Reply:
Django doesn't support cross-database queries for now.